Sources/ExyteChat/ChatView/ChatView.swift

```diff
@@ -184,19 +188,61 @@ public struct ChatView<MessageContent: View, InputViewContent: View, MenuAction:
                     .ignoresSafeArea()
                 }
             }
-
+            .onAppear() {
+              if isGiphyAvailable() {
+                if let giphyKey = giphyConfig.giphyKey {
+                  if !giphyConfigured {
+                    giphyConfigured = true
+                    Giphy.configure(apiKey: giphyKey)
+                  }
+                } else {
+                  print("giphy input available though giphy key not provided")
```

Comment:
Since this is the official error handling process here, let's make it more beautiful: "WARNING: giphy key not provided, please pass a key using giphyConfig"

Reply:
will change